### ISSUE TITLE
Code Linting

### ISSUE THREAD

**Issue by irmerk** [bookworm-reads/documentation#23]
We need linting and a coding style guide for each service

## Use Case
Our code should be more accessible by being consistent in style

## Possible Solution
- https://github.com/bookworm-reads/server
- https://github.com/bookworm-reads/mobile
- https://github.com/bookworm-reads/web
- https://github.com/bookworm-reads/documentation (if appropriate)

## Context
Ensure each service has appropriate linting rules for the code inside - such as React rules for web, but not for server
